### PR TITLE
Maternity, paternity and shared leave

### PR DESCRIPTION
**Pull request by RoryMacDonald**
Some thoughts on maternity / paternity / shared parental leave

### CODE REVIEW

**Comment by idabmat** [comment]
This is much clearer :+1:

**Comment by lukemorton** [comment]
Although I think this is what we currently offer I don't think it's a fair package. Seeing as we already allow staff to take 4 weeks holiday we could at least offer 4 weeks full pay if not 6.

File: guides/welfare/parental_leave.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
-# Parental Leave
+# Maternity, Paternity & Shared Parental Leave
 
-Our current parental leave package is two weeks at full pay followed by statutory allowance.
+When you welcome a new child (birthed or adopted), Made Tech encourages you to take leave, to welcome the child into the world.
```
Comment (sowasred2012) [inline guides/welfare/parental_leave.md]: Lots of "welcoming" - maybe:
When you welcome a new child (birthed or adopted), Made Tech encourages you to take leave and spend time introducing your child to the world.
?

**Comment by sowasred2012** [comment]
Not sure how much of this is down to Government mandate or a per company basis - but would love to see the UK adopt [the Swedish approach](https://www.theguardian.com/world/2015/may/28/swedish-fathers-paid-paternity-parental-leave).

File: guides/welfare/parental_leave.md (hunk 1 above)
Comment (RyanMacG) [inline guides/welfare/parental_leave.md]: I'd recommend `introducing your child to the family` or something, adopted children generally older so have already see the world a bit. Just a thought

Comment (RyanMacG) [inline guides/welfare/parental_leave.md]: For context:

> 5% (3,540) of children looked after on 31st March 2016 were under 1 year old
13% (9,140) were aged between 1 and 4 years old
20% (14,090) were aged between 5 and 9 years old
39% (27,220) were aged between 10 and 15 years old
23% (16,460) were aged 16 and over

From - https://www.adoptionuk.org/press-media/adoption-facts-and-figures-england

**Comment by chrisblackburn** [comment]
@lukemorton r.e. the holiday pay, I think you still accrue holiday allowance while on leave (https://www.nidirect.gov.uk/articles/entitlements-during-statutory-maternity-leave)

**Comment by lukemorton** [comment]
@chrisblackburn Yeah that would be the additional cost to the business compared to using holiday instead of mat/pat leave.

**Pull request merged**